docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -46,6 +46,37 @@ quarkus.http.auth.policy.role-policy1.roles-allowed=user,admin
 ----
 <1> This defines a role-based policy that allows users with the `user` and `admin` roles.
 
+Sometimes it might be useful to register your own named policy. You can get it done by creating application scoped CDI
```

Comment:
Would make sense to have a dedicated `Custom HttpSecurityPolicy` IMHO

Reply:
Done, let's see if it's better.